src/pages/index.js

```diff
@@ -68,51 +98,138 @@ cardPreviewPopup.close();
 /*                                Card Section                                */
 /* -------------------------------------------------------------------------- */
 
+
 // This function creates a new card
-function createCard(data) {
-    const cardElement = new Card({ data, handleImageClick: (imageData) => {
-        cardPreviewPopup.open(imageData);
+function createCard(data, userId) {
+    const cardElement = new Card({ 
+        data,
+        handleImageClick: (imageData) => {
 
-    }}, selectors.cardTemplate);
+            // Open image popup on click
+            cardPreviewPopup.open(imageData);
+        },
+        handleDeleteClick: () => {
+            
+            // Create the delete popup instance
+            const deletePopup = new PopupWithConfirm(selectors.deletePopup, () => {
```

Comment:
There should be only one instance of this class

Reply:
Certainly! That's fixed. Thank you.